Italian/guardare/vecchie-fotografie/vecchie-fotografie.md

```diff
@@ -5,3 +5,7 @@ Sembrano fotografie scattate negli anni 50, almeno a giudicare dall'abbigliament
 
 Vi sono uomini e donne di diversa età, e in alcune immagini, inspiegabilmente, vi sono delle bruciature sul viso 
 di alcune persone.
+
+[Sfogli i libri, in cerca di qualche indizio](libri-francesi/libri-francesi.md)
```

Comment:
@Gus42, based on your description, I think maybe this is intentional and you are hoping someone else will add this page. However, I don't want the story to become full of dead links. Instead, could you either remove this option, or create a page with a sentence or two for it to lead to? Then other people can add their own options / continuations.